src/views/TradingCompetition/components/Section/index.tsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react'
+import styled from 'styled-components'
+import { Flex } from '@pancakeswap-libs/uikit'
+import { SectionProps } from '../../types'
+import IntersectionCurve from './IntersectionCurve'
+
+const BackgroundColorWrapper = styled(Flex)<SectionProps>`
+  position: relative;
+  flex-direction: column;
+  z-index: ${({ index }) => index};
+  background: ${({ backgroundStyle }) => backgroundStyle};
+
+  padding-top: ${({ intersectionPosition }) => (intersectionPosition === 'top' ? '6px' : '48px')};
+  margin: ${({ intersectionPosition }) => (intersectionPosition === 'top' ? '0' : '-34px')} 0;
+
+  ${({ theme }) => theme.mediaQueries.sm} {
+    margin: ${({ intersectionPosition }) => (intersectionPosition === 'top' ? '0' : '-42px')} 0;
+  }
+`
+
+const ChildrenWrapper = styled.div`
```

Comment:
This component essentially applies the `Page` (and by extension `Container`) styles minus the min-height to the sections' `ChildrenWrapper`.

The min-height is applied to the `TradingCompetition` index wrapper.

Not sure if there's a more 'scalable' solution here, or if you're happy with this?

Reply:
You wrote it yourself, it uses the Page/Container style, then why would you duplicate the content here

Reply:
Yea man I can just overwrite the Page style that I don't want. My brain must have been much when I did this initially